src/server/adminBro.ts

```diff
@@ -2287,6 +2292,10 @@ export const verifyProjects = async (
         }
       }
     }
+
+    await refreshUserProjectPowerView();
+    await refreshProjectPowerView();
+    await refreshProjectFuturePowerView();
```

Comment:
I would do it in one line
```
await Promise.all([
    refreshUserProjectPowerView(),
    refreshProjectPowerView(),
    refreshProjectFuturePowerView()
])
```

Reply:
Thanks @aminlatifi , you are absolutely right, I did it.